src/fragmentarium/ui/image-annotation/annotation-tool/Help.tsx

```diff
@@ -11,8 +11,7 @@ export default function Help(): JSX.Element {
             <b>Esc</b> Reset
           </li>
           <li>
-            <b>Ctrl + Select Annotation:</b> Change Reading on existing
-            Annotation
+            <b>y + Select Annotation:</b> Change Reading on existing Annotation
```

Comment:
Why `y`, feels less natural than control?

Reply:
can we leave it like that for now right click is problematic because it looks like i would have to "catch" the right click and return the coordinates in relative fashion to the internal annotation handler for click.